server/migrations/20220607213602-terra-token-threshold.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.bulkUpdate('ChainNodes', {
+      url: 'https://lcd.phoenix.terra.setten.io/5e351408cfc5460186aa77ff1f38fac9',
```

Comment:
We use the tendermint rather than lcd url for cosmos chains, so we should very clearly document the usage of lcd here, and also this will break the client-side adapter if we try to implement any cosmos functionality.